pkg/config/schema.json

```diff
@@ -327,9 +327,6 @@
           },
           "type": "array"
         },
-        "os-release": {
```

Comment:
I didn't touch this but after running `make generate` this and the `OSRelease` below were removed.